bin/node/cli/src/service.rs

```diff
@@ -715,15 +741,26 @@ where
 		grandpa_block_import,
 		client.clone(),
 	)?;
-	let inherent_data_providers = InherentDataProviders::new();
+	let slot_duration = babe_link.config().slot_duration();
 	// FIXME: pruning task isn't started since light client doesn't do `AuthoritySetup`.
 	let import_queue = sc_consensus_babe::import_queue(
 		babe_link,
 		babe_block_import,
 		Some(Box::new(justification_import)),
 		client.clone(),
 		select_chain.clone(),
-		inherent_data_providers.clone(),
+		move |_, ()| async move {
+			let timestamp = sp_timestamp::InherentDataProvider::from_system_time();
+			let slot =
+				sp_consensus_babe::inherents::InherentDataProvider::from_timestamp_and_duration(
+					*timestamp,
+					slot_duration,
+				);
+			let uncles =
+				sp_authorship::InherentDataProvider::<<Block as BlockT>::Header>::check_inherents();
+
+			Ok((timestamp, slot, uncles))
```

Comment:
Compared with https://github.com/paritytech/polkadot/tree/master/node#L1156 `Ok((timestamp, slot))`

The return value should not contain `uncles`?

Reply:
Because Polkadot doesn't require this `Inherents`